### DISCUSSION TITLE
Welcome to the internationalization team!

### DISCUSSION THREAD

**Discussion by tesseralis** [reactjs/reactjs.org-translation#226] [Team Posts]
Use this team to discuss general thoughts/questions on maintaining the different translation forks(e.g. "should external links be localized").

**Comment by smikitky** [comment]
Regarding external links: Most of the time, we simply cannot localize. Most links which *are* technically localizable are articles of either MDN (e.g. 'arrow functions') or Wikipedia (e.g., 'memoization'), but then, these pages already have good "switch language" UI. So from the practical point of view, the usefulness of changing a small fraction of URLs is small. But as I said earlier, I just wanted to wait for some consensus, so if other maintainers are fine with changing URLs, I'm fine too.

**Comment by tesseralis** [comment]
@potato4d can you reiterate your experience with the jp.vuejs.org docs?

@QC-L what did you do for the Chinese React docs?

**Comment by potato4d** [comment]
@tesseralis 

As commented on the [issues/4](https://github.com/reactjs/ja.reactjs.org/issues/4#issuecomment-459578301), it is as follows!

>  In the case of Vue document, when I find a document translated by MDN etc., I am changing the link.
> 
> For example, most people reading Japanese documents are Japanese.
> And, in most cases the Japanese are seeking Japanese document.

**Comment by alejandronanez** [comment]
Hey friends,

We're localizing external links in the Spanish repo. If the translated article is not there yet, MDN will default to the English version (that's what I've noticed so far)

**Comment by JeremiePat** [comment]
@alejandronanez Regarding links to MDN,if you skip the local from the URL, MDN will guess the best local from the HTTP headers sent by your browser then fallback to English if the content hasn't been localized yet.

**Comment by alejandronanez** [comment]
I didn't know that @JeremiePat, thanks for the tip!

**Comment by gaearon** [comment]
Let's use https://github.com/reactjs/reactjs.org-translation/issues for this instead now.
